### ISSUE TITLE
delete(E entity) vs delete(Iterable<E> entities)

### ISSUE THREAD

**Issue by imort** [requery/requery#406]
I should say thank you @npurushe for hard work on this library. Please look at this:
```
@Override
    public <E extends T> Void delete(E entity) {
        try (TransactionScope transaction = new TransactionScope(transactionProvider)) {
            EntityProxy<E> proxy = context.proxyOf(entity, true);
            synchronized (proxy.syncObject()) {
                context.write(proxy.type().getClassType()).delete(entity, proxy);
                transaction.commit();
            }
        }
        return null;
    }

    @Override
    public <E extends T> Void delete(Iterable<E> entities) {
        Iterator<E> iterator = entities.iterator();
        if (iterator.hasNext()) {
            try (TransactionScope transaction = new TransactionScope(transactionProvider)) {
                E entity = iterator.next();
                EntityProxy<E> proxy = context.proxyOf(entity, false);
                EntityWriter<E, T> writer = context.write(proxy.type().getClassType());
                writer.delete(entities);
                transaction.commit();
            }
        }
        return null;
    }
```

I have noticed that in group deletion we do not notify updates for entity type so we do not trigger observableResult() update. Is there any reason behind this behaviour?

I expected observableResult() be triggered on group deletion, am I missing something?

**Comment by npurushe** [comment]
Thanks for reporting, both types of deletions should trigger an update

**Comment by npurushe** [comment]
This fix is in version 1.2.0 thanks
